components/styles/GlobalStyles.js

```diff
@@ -0,0 +1,63 @@
+import { createGlobalStyle } from 'styled-components';
+
+const GlobalStyles = createGlobalStyle`
+:root {
+  box-sizing: border-box;
+  --Body: black;
```

Comment:
Nitpick: I see the last two css variables you've used only lowercase. Stay consistant; either start all with or without uppercase.
Personally I prefer lowercase (so camelcase)